packages/dd-trace/test/util.spec.js

```diff
@@ -17,6 +17,22 @@ const FALSES = [
   'fAlSe'
 ]
 
+const MATCH_CASES = [
```

Comment:
As we'll be moving more towards integration tests, could these be tested using the sampler instead?

Reply:
Systems tests will cover that, so not sure we need our own integration tests.

Reply:
My point was more about removing the unit tests, then if we have existing integration tests nothing more needs to be added.